mochify/lib/load-config.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+
+const { cosmiconfig } = require('cosmiconfig');
+
+exports.loadConfig = loadConfig;
+
+async function loadConfig(config) {
+  const explorer = cosmiconfig('mochify', {
+    searchPlaces: [
+      'package.json',
+      '.mochifyrc',
```

Comment:
Just curious: what' the expected format for a naked rc file? JSON? Or would YAMLwork too?

Reply:
The cosmiconfig docs say it can be YAML or JSON:

> YAML is a superset of JSON; which means YAML parsers can parse JSON; which is how extensionless files can be either YAML or JSON with only one parser.